lib/lib8tion/lib8tion.h

```diff
@@ -167,8 +167,8 @@ Lib8tion is pronounced like 'libation': lie-BAY-shun
 
 #include <stdint.h>
 
-#define LIB8STATIC __attribute__ ((unused)) static inline
-#define LIB8STATIC_ALWAYS_INLINE __attribute__ ((always_inline)) static inline
+#define LIB8STATIC
```

Comment:
Wha was this changed?

Reply:
Firmware size decrease. The unused attribute one at least was causing functions to be included in the firmware that were unused.

Reply:
okay.

Reply:
backed out this change, will do this in a separate pr.